### PR TITLE
Improve 'Elsewhere' links on location pages

### PR DESCRIPTION
**Pull request by aduggin**

### CODE REVIEW

File: app/views/locations/show.html.erb [hunk 1]
```diff
@@ -45,3 +45,23 @@
 <p>
   You can also book in person by visiting any Citizens Advice bureau.
 </p>
+
+<% content_for :sidebar do %>
```
Comment (stevenwilkin) [inline app/views/locations/show.html.erb]: Any reason for moving the `sidebar` content block out of `layouts/locations.html.erb` and into this view?

File: app/controllers/guides_controller.rb [hunk 2]
```diff
@@ -48,7 +48,12 @@ def decorate(guide_or_collection)
   end
 
   def non_journey_related_guides
-    @related_guides ||= guide_repository.find_all(*NON_JOURNEY_RELATED_GUIDE_IDS)
+    @related_guides ||= begin
+      ids = *NON_JOURNEY_RELATED_GUIDE_IDS
```
Comment (stevenwilkin) [inline app/controllers/guides_controller.rb]: Can just use ``ids = NON_JOURNEY_RELATED_GUIDE_IDS``, don't need the splat.

File: app/controllers/guides_controller.rb [hunk 3]
```diff
@@ -48,7 +48,12 @@ def decorate(guide_or_collection)
...
+      ids.reject! { |id| id == 'appointments' } if params[:id] == 'book'
```
Comment (stevenwilkin) [inline app/controllers/guides_controller.rb]: There's no tests for this.

File: app/views/locations/show.html.erb (hunk 1 above)
Comment (andrewgarner) [inline app/views/locations/show.html.erb]: Because it is only required when viewing a location, not for all other location pages.

Comment (stevenwilkin) [inline app/views/locations/show.html.erb]: Fair enough!

**Pull request merged**